src/sidebar/components/search/test/FilterAnnotationsStatus-test.js

```diff
@@ -69,7 +69,7 @@ describe('FilterAnnotationsStatus', () => {
     });
   });
 
-  context('(State 1): no search filters active', () => {
+  context('no search filters active', () => {
```

Comment:
The state numbering either needed to be redone or removed here. I just removed it.